code/modules/admin/verbs/anonymousnames.dm

```diff
@@ -0,0 +1,67 @@
+/client/proc/anon_names()
+	set category = "Admin - Events"
+	set name = "Setup Anonymous Names"
+
+
+	if(SSticker.current_state > GAME_STATE_PREGAME)
+		to_chat(usr, "This option is currently only usable during pregame.")
+		return
+
+	if(SSticker.anonymousnames)
+		SSticker.anonymousnames = ANON_DISABLED
+		to_chat(usr, "Disabled anonymous names.")
+		message_admins("<span class='adminnotice'>[key_name_admin(usr)] has disabled anonymous names.</span>")
+		return
+	var/list/names = list("Cancel", ANON_RANDOMNAMES, ANON_EMPLOYEENAMES)
+	var/result = input(usr, "Choose an anonymous theme","going dark") as null|anything in names
```

Comment:
note this is not a stripped input

just sayin

Reply:
??? it's from the list

Reply:
sorry im r*******

thanks for helping